lib/api.js

```diff
@@ -35,6 +35,8 @@ module.exports = settings => {
 
   app.use('/establishment(s)?', require('./routers/establishment'));
 
+  app.use('/pil/training', require('./routers/trainingModules'));
```

Comment:
I'm not sure this is the best routing structure.

I'd be tempted to scope the training modules underneath a `/profile/:profileId` router, so that the routes more accurately represented the data structures - that is that a training module is a sub-property of a profile.

Reply:
I created https://jira.digital.homeoffice.gov.uk/browse/AS-662.
Merging.